src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier extends Figure {
```

Comment:
FigureSuplier is just a class for business logic purposes, we use it to get some figures. Why does it extend Figure? FigureSupplier doesn't want to have Figure state and functionality. Now FigureSupplier inherited color and a couple of methods